webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/TableIndexPage.js

```diff
@@ -237,10 +257,12 @@ const TableIndexPage = ({
 
             {total > 0 && (
               <Pagination
+                key="table-index-page-top-pagination"
+                updateParamsByUrl={updateParamsByUrl}
                 variant={PaginationVariant.top}
-                page={page}
-                perPage={perPage}
-                itemCount={subtotal}
+                page={Number(page) || 1}
+                perPage={Number(perPage) || 20}
```

Comment:
since this is repeating 3 times in this file can it be in const in case we ever change it?

Reply:
Meant also the page || 1.
Also actually why is this needed? the default value for page is 1 and perPage uses the settings perPage if not defined